pkg/reconciler/shared/tektonconfig/tektonconfig.go

```diff
@@ -231,6 +232,12 @@ func (r *Reconciler) markUpgrade(ctx context.Context, tc *v1alpha1.TektonConfig)
 		tc.Status.MarkPostInstallFailed(v1alpha1.UpgradePending)
 		tc.Status.MarkNotReady("Upgrade Pending")
 	}
+
+	// TODO: remove later, as it is for handling specific case of one upgrade
+	if ok && !strings.HasPrefix(ver, getPatchVersionTrimmed(r.operatorVersion)) {
+		tc.Spec.Pipeline.EmbeddedStatus = "both"
```

Comment:
later time user may change the value, but I guess this logic will reset the `tc.Spec.Pipeline.EmbeddedStatus` to `both`, in the future release. example: `0.69 != 0.70`

Reply:
This a lag which we introduced in 0.62 and changing the value in 0.65, and this will be removed in 0.67 i guess

Reply:
Its a short term thing, we need to do otherwise the v1 resources are not working as expected after upgrade

Reply:
may be we can add check like if the value is `full` the only do change, otherwise not